java/NpyWriter.java

```diff
@@ -11,7 +11,7 @@
 
 public class NpyWriter {
 
-    private static final Boolean COMPRESS = true;
+    private static final boolean COMPRESS = false;
```

Comment:
Is this for processing speed reasons?
In case we want to keep the temporary .npy file (see first python comment) we would probably want it compressed. Will of course trust your good judgement on this.

Reply:
Thanks Aiden. Yes, fully for performance reasons. When I had implemented the compression I had included it as a default when `close()`ing. In hindsight, I think this is strange and I'll change it to make compression explicit.